core/src/com/biglybt/core/networkmanager/impl/tcp/SelectorGuard.java

```diff
@@ -137,8 +138,9 @@ public void verifySelectorIntegrity( int num_keys_ready, long time_threshold ) {
 
 
     if( consecutiveZeroSelects > SELECTOR_FAILURE_THRESHOLD ) {  //should only happen under Windows + JRE 1.4
```

Comment:
Is ther a flaw in the semantics here preventing this if-block ever to be reached?
The previous if-block takes precedence. Would make sense if they switched.

Reply:
If the block is never reached given current code paths then go ahead and remove it (and check implications of failureDetected() now never being called if you want)

Reply:
See #1005